tf2onnx/tfonnx.py

```diff
@@ -1809,7 +1809,7 @@ def tf_optimize(inputs, outputs, graph_def, fold_constant=None):
 
 def process_tf_graph(tf_graph, continue_on_error=False, verbose=False, target=None,
                      opset=None, custom_op_handlers=None, custom_rewriter=None,
-                     extra_opset=None, shape_override=None, inputs_as_nchw=None):
+                     extra_opset=None, shape_override=None, inputs_as_nchw=None, output_names=None):
```

Comment:
since we pass in output_names maybe it should be required and removed from make_model() and once we do this we should include inputs as well?

Reply:
the argument-"inputs" is not used in the code.

we could find the inputs from the graph >> if the op type is "Placeholder" then we treat it as a input.
another thought is that: sometimes we only want the result of internal node, then maybe not all inputs of graph are needed, so we have to find the inputs by ourselves

Reply:
fair.